packages/core/admin/admin/src/pages/MarketplacePage/tests/plugins.test.js

```diff
@@ -20,9 +20,8 @@ import MarketPlacePage from '../index';
 import server from './server';
 
 const toggleNotification = jest.fn();
-
 jest.mock('../../../hooks/useNavigatorOnLine', () => jest.fn(() => true));
-
+jest.setTimeout(30000);
```

Comment:
ℹ️ I don't see any other option here I think the tests just take longer than the 5000ms default.

I tried passing it as a second argument to the tests concerned which works when running this test suite but throws an error when running with all the Strapi tests

Reply:
I think it's okay, eventually we'll have to find a way to make the tests faster but for now this is needed